### PR TITLE
updated novo tiles to have a disable option

### PR DESCRIPTION
**Pull request by rmcdole**
##### **Reviewers**
* @jgodi
* @more

### CODE REVIEW

File: src/elements/tiles/Tiles.ts [hunk 1]
```diff
@@ -43,6 +43,7 @@ export class NovoTilesElement implements ControlValueAccessor, OnInit {
     @Input() options: any;
     @Input() required: boolean;
     @Output() onChange: EventEmitter<any> = new EventEmitter();
+    @Input() enabled: boolean = true;
```
Comment (jgodi) [inline src/elements/tiles/Tiles.ts]: Can you flip this, make the input `disabled`

File: src/elements/tiles/Tiles.scss [hunk 2]
```diff
@@ -16,6 +16,10 @@ novo-tiles {
         cursor: pointer;
         align-items: center;
 
+        &.disabled {
+            opacity: .5;
```
Comment (jgodi) [inline src/elements/tiles/Tiles.scss]: pointer-events: none; and then you dont need the check in the select

Comment (rmcdole) [inline src/elements/tiles/Tiles.scss]: thats disabled clicking on it, which prevents our toast from happening.

Comment (jgodi) [inline src/elements/tiles/Tiles.scss]: No, I mean if its disabled, then you probably shouldn't want to click on it at all. You have it:

```
        if (this.disabled){
            this.onChange.emit(item.value);
            this.onModelChange(item.value);
            return;
        }
```

which will stilll emit the change event for something that didn't get reflected in the UI.

if you just modify the CSS to have `pointer-events: none` then you won't need the logic above

Comment (rmcdole) [inline src/elements/tiles/Tiles.scss]: ah, i see what you are saying. sounds like we should rename the variable instead then to make it more clear. ill think of something :thinking:

Comment (rmcdole) [inline src/elements/tiles/Tiles.scss]: do you have any ideas?

Comment (rmcdole) [inline src/elements/tiles/Tiles.scss]: Okay I added the toast logic to this class to hopefully make this more clear. If disabled, everything is disabled except toasts if toastOptions are passed in. Do you think the var name still needs to be changed?

**Pull request merged**